src/gerbil/runtime/gx-gambc1.scm

```diff
@@ -109,7 +109,7 @@
                             (current-readtable)
                             _gx#wrap-syntax
                             _gx#unwrap-syntax
-                            #f #f))))
+                            #f '() #f))))
```

Comment:
We want to cond-expand for the version here (look at eval-when in gx-gambc0) so that it continues working with the latest release.
We can also get away with a runtime-check of the version using `system-version`, that's probably easier (albeit less efficient).
I don't want to have master broken for the last release.

Reply:
The easiest way forward is probably to abstract the call into a readenv constructor that is cond-expand protected with `eval-when`.

Reply:
It looks like the gambit master branch does not update its version, so we need to wait for gambit to adopt a new version number.

Reply:
That's unfortunate. There has to be some hack we can do to detect when to use the extra argument.

Reply:
Perhaps, as a transient measure, do a call with the old signature and have an exception handler that handles arity errors and calls with the new signature?
That could work as a workaround until there is a new gambit release.